starter_project_mobile_2022/lib/add_blog/constants/constants.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+const pageBackground = Colors.white54;
+
+const titleFontSize = 20;
+const titleFontWeight = FontWeight.bold;
+const titleFontColor = Colors.black;
+
+const tagFontSize = 18;
+const tagFontWeight = FontWeight.normal;
+const tagFontColor = Colors.white;
+
+const buttonBackgroundColor = Colors.blue;
+const buttonContentColor = Colors.white;
+const buttonHeight = 50;
+const buttonWidth = 200;
+const double buttonBorderRadius = 60;
+const buttonTextFontWeight = FontWeight.bold;
+
+const menuIcon = Icons.menu;
+const addTagButtonIcon = Icons.add_circle_outline_outlined;
+const addImageIcon = Icons.photo_camera;
+const addVideoIcon = Icons.video_library;
+const attachFilesIcon = Icons.attachment;
+const modifyAlignmentIcon = Icons.format_align_center;
+const modifyTextIcon = Icons.text_fields;
+const clearIcon = Icons.clear;
+
+const outerWidgetPadding = EdgeInsets.symmetric(horizontal: 25, vertical: 10);
```

Comment:
Using hardcoded constants for dimensions will cause problems. 
We should use something like;
```
double width = MediaQuery.of(context).size.width;
EdgeInsets.symmetric(horizontal: width * 0.002, vertical: width * 0.001); 
```
anytime we need dimensions.